superset/databases/commands/exceptions.py

```diff
@@ -130,7 +130,8 @@ class DatabaseTestConnectionDriverError(CommandInvalidError):
     message = _("Could not load database driver")
 
 
-class DatabaseTestConnectionUnexpectedError(CommandInvalidError):
+class DatabaseTestConnectionUnexpectedError(SupersetErrorsException):
+    status = 422
```

Comment:
Where could I learn what status codes signify what?

Reply:
I use the Mozilla docs for reference, unless we've got custom status codes:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#client_error_responses

Reply:
https://httpstatuses.com is good too